fury/ui.py

```diff
@@ -3529,7 +3530,8 @@ def __init__(self, values, position=(0, 0), size=(100, 300),
                  unselected_color=(0.6, 0.6, 0.6),
                  scroll_bar_active_color=(0.6, 0.2, 0.2),
                  scroll_bar_inactive_color=(0.9, 0.0, 0.0),
-                 background_opacity=1.):
+                 background_opacity=1., directory_path=None,
```

Comment:
I don't understand why `ListBox2D` needs a `directory_path` and `extensions` variables. That should go/stay in `FileMenu2D` or, if needed, in a subclass of `ListBox2D` to have a specialized list, e.g. `FileListBox2D` or something.

Reply:
Hey @MarcCote! Thanks for reviewing! 
I understood your point in this comment, but your previous [comment](https://github.com/fury-gl/fury/pull/222#pullrequestreview-384797475) is not so clear. Can you please explain a bit more?

Reply:
I wrote the comment you are referring to while trying to remember what was the issue raised by #16 (possibly outdated). Here, I think I found a better alternative that leverages `ListBox2D.on_change` callback to change directory.
https://github.com/devmodi154/fury/pull/1

Caveat: it's not dealing with multi-selection (I just take/look at the first selected item), if it is a directory, then move to it. We probably need a double-click event instead (not implemented yet - a similar idea as for `UI.on_left_mouse_button_dragged` might be doable).

Reply:
@skoudoro we might need a double-click callback ^

Reply:
`might` :smile: I agree, I already had the need for this feature.  I just created an issue as a reminder (#226)

Reply:
Thanks for explaining @marscos! The scrolling refactor issue has been resolved but I had a doubt, how about we make ScrollBar a separate component? 
As scrolling can be needed for many other components. Example: A text box has a large text.

Reply:
Good catch. I like the idea of having a ScrollBar component. :)

Reply:
@MarcCote I have tried to separate the ScrollBar component. I wanted to know two things :
- Is this a good direction for the idea of ScrollBar class.
- Would we want the functionality of scrolling using mouse events inside the ScrollBar class or should it belong to list itself?

Reply:
It could be a good thing, yes. I see you have made an attempt at it. I'll look at it this week.